CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` would be raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+- `Session.create_dataframe()` no longer ignores the optional parameter `schema` when `data` is a pandas DataFrame.
+
+### New Features:
+- Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` in functions.
+- Enabled creating UDFs that reference non-local functions. This feature is not available inside stored procedures.
+- Added the property `DataFrame.dtypes` to return a list of column name, data type pairs.
+- Added the following aliases:
+  - `functions.expr()` for `functions.sql_expr()`.
+  - `functions.date_format()` for `functions.to_date()`.
+  - `functions.monotonically_increasing_id()` for `functions.seq8()`
+  - `functions.from_unixtime()` for `functions.to_timestamp()`
+
+### Bug Fixes:
+- Fixed a bug in SQL simplifier that didn’t handle Column alias and join well in some cases.
+- Fixed a bug where `Session.create_dataframe` ignored user-specified `schema` and used inferred schema when the input data is a pandas DataFrame.
```

Comment:
Is this a duplicate with the one in behavior changes?

Reply:
Yes, the bug fix is a behavior change, idk if we should put them just in one section? I worded the two items differently.

Reply:
I prefer one section